src/testing/rapid_tests.py

```diff
@@ -53,3 +80,25 @@ def _test_schools_and_educ_workers(states, contacts):
     too_long_since_last_test = states["cd_received_rapid_test"] <= -3
     to_test = eligible & too_long_since_last_test & has_educ_contacts
     return to_test
+
+
+def _give_rapid_tests_to_some_workers(states, contacts, share_workers_to_be_tested):
+    date = get_date(states)
+    work_cols = [col for col in contacts if col.startswith("work_")]
+    has_work_contacts = (contacts[work_cols] > 0).any(axis=1)
+
+    # starting 2021-04-26 every worker must be offered two tests per week
+    # source: https://bit.ly/2Qw4Md6
+    if date > pd.Timestamp("2021-04-26"):
+        too_long_since_last_test = states["cd_received_rapid_test"] <= -3
+    else:
+        # Assume that workers are only tested weekly.
+        too_long_since_last_test = states["cd_received_rapid_test"] <= -7
+
+    should_get_test = has_work_contacts & too_long_since_last_test
+    truth_probabilities = np.full(len(states), share_workers_to_be_tested)
+    receives_offer_and_accepts = pd.Series(
```

Comment:
In the current version too many tests are demanded because the compliance is not correlated over time. Thus someone who should have made a test today and did not get's a new chance tomorrow. 

We can use a time persistent test compliance column in the states to solve this. This is probably also a good assumption because both test offers from employers and test demand from employees is highly correlated over time.

Reply:
Implemented this in `a71c7c`.